src/CMakeLists.txt

```diff
@@ -7,13 +14,16 @@ add_library(
     cps/search.cpp
     cps/utils.cpp
     cps/version.cpp
+    ${BISON_PcParser_OUTPUTS}
+    ${FLEX_PcScanner_OUTPUTS}
```

Comment:
*Sigh*

It seems probable that this is how the CMake modules for Bison and Flex were designed, but for the record, it's gross to expand strings into add_library calls instead of having a `target_bison_codegen` function that just adds the source files to your target for you. The function should also be adding the needed include directory for you so you don't have to add "magic" include paths as you do below.

Let's circle back with Kitware about whether the flex and bison codegen need another look, especially now that CMake 3.31 will support code generation better than before.

Reply:
Just to close the feedback loop, this is still something that needs cleaned up if someone's looking for a good upstream CMake contribution to start with. But it wasn't scoped into any work yet. Using the API like this is the best thing we can do for now.